packages/image_picker/image_picker/example/lib/main.dart

```diff
@@ -139,7 +143,11 @@ class _MyHomePageState extends State<MyHomePage> {
       return retrieveError;
     }
     if (_imageFile != null) {
-      return Image.file(File(_imageFile.path));
+      if (kIsWeb) {
+        return Image.network(_imageFile.path);
```

Comment:
nits: Add a comment explaining why we need to use .network for web.

Reply:
I'm going to point people to the README of the plugin, where this is already explained:

https://pub.dev/packages/image_picker#getting-ready-for-the-web-platform

Reply:
Done